Unit_testing_frameworks/Calculator_Tests/SqrtTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using CSharpCalculator;
+
+namespace Calculator_Tests
+{
+	[TestClass]
+	public class SqrtTests
+	{
+        [TestMethod]
+        public void SqrtNegativeNumber()
+        {
+            // Arrange
+            double param = -100;
+            double expectedRes = double.NaN;
+            Calculator calc = new Calculator();
+
+            // Act
+            calc.Sqrt(param);
```

Comment:
unnecessary method call

Reply:
answered below. if I'm wrong I'll fix it:)